diesel_codegen/Cargo.toml

```diff
@@ -10,7 +10,7 @@ repository = "https://github.com/diesel-rs/diesel/tree/master/diesel_codegen"
 keywords = ["orm", "database", "postgres", "sql", "codegen"]
 
 [dependencies]
-diesel_codegen_syntex = { version = "0.7.0", default-features = false }
+diesel_codegen_syntex = { path = "../diesel_codegen_syntex", version = "0.7.3", default-features = false }
```

Comment:
Even with the local override, this is required because version 0.7.3 hasn't been published to crates.io. From the [Cargo documentation](http://doc.crates.io/specifying-dependencies.html#overriding-with-local-dependencies):

> Note: using a local configuration to override paths will only work for crates that have been published to crates.io. You cannot use this feature to tell Cargo how to find local unpublished crates.